CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Added
+
+#### From [#605](https://github.com/polkadot-fellows/runtimes/pull/605)
```

Comment:
I think this "From ..." notation makes sense when there are many merge requests, just for two it is fine to note it at the end of the line.

Reply:
Got it.